cpp/ps0003/p3Solution.cpp

```diff
@@ -9,11 +9,13 @@ void checkEvenorOdd(int num) {
 }
 
 int main() {
+	// prompting user for number 2/2
 	int num;
 	cout << "Enter a number to check if it's even or odd: ";
 	cin >> num;

```

Comment:
- [x] declared a variable to store users input
- [x] used `std::cin` to prompt user for the number